File: Time.pl
my $freq = 120;
my $sec_in_min = 60;
$freq /= $sec_in_min;

my $o;

my $t = <>;
 
if ($t > 0) {
    $o = $freq / $t;
} else {
    $o = -1;
}
 
say $o;

# Code by GPU Officer Yahwe

File: Time.pas
Program Time;
Const 
   Freq = 120;
   Sec_in_min = 60;
Var 
   o, t : Real;
   f : Real;
Begin
   f := Freq / Sec_in_min;
   ReadLn(t);
   if (t > 0) 
      then
          o := f / t;
      else 
          o := -1;
      end;

   WriteLn(o);
End.

{ Code by GPU Officer Yahwe, Toverisch Trunaev, Russia } 
